### ISSUE TITLE
Logrotate does not include passenger-config repopen-logs

### ISSUE THREAD

**Issue by groyoh** [phusion/passenger-docker#255]
Hello,

My team recently tried to upgrade from rel-0.9 to rel-1.0 and we faced an issue with the logs where NGINX ended up writing logs to `error.log.1` rather than `error.log`. After some investigation, I realized that the logrotate configuration for 0.9 contained this `postrotate` script:

```
	postrotate
		sv 1 nginx >/dev/null 2>&1 || true
		passenger-config reopen-logs >/dev/null 2>&1 || true
	endscript
```

while the 1.0 one contained only:

```
	postrotate
		sv 1 nginx >/dev/null 2>&1 || true
	endscript
```

So NGINX is not reopening the logs in 1.0.

After more digging, I understand that it comes from the fact that starting 1.0, passenger-docker uses the official NGINX deb rather than the one built from https://github.com/phusion/passenger_apt_automation (correct me if I'm wrong there). And therefore, this [file](https://github.com/phusion/passenger_apt_automation/blob/master/debian_specs/nginx/nginx-common.nginx.logrotate.erb) which contains the `passenger-config reopen-logs` is no longer copied to logrotate config folder.

I solved this issue by adding this missing line to our logrotate config but I wondered whether this was a gotcha that was documented somewhere and I missed it or whether it's a bug.

**Comment by CamJN** [comment]
This was an oversight in the 1.0 release it will be fixed in the upcoming release.

**Comment by acejam** [comment]
I'm also seeing this, and when it happens, pretty much all output from `docker logs` stops, causing our downstream log processor to think the app is no longer capturing logs.

**Comment by groyoh** [comment]
@acejam if it can help you, our temporary solution was to create a file with this content:

```
/var/log/nginx/*.log {
	daily
	rotate 14
	missingok
	compress
	delaycompress
	notifempty
	create 0640 www-data adm
	sharedscripts
	prerotate
		if [ -d /etc/logrotate.d/httpd-prerotate ]; then \
			run-parts /etc/logrotate.d/httpd-prerotate; \
		fi \
	endscript
	postrotate
		sv 1 nginx >/dev/null 2>&1 || true
		passenger-config reopen-logs >/dev/null 2>&1 || true
	endscript
}
```

And added this in our Dockerfile:
```
ADD delivery/logrotate-nginx /etc/logrotate.d/nginx
```

**Comment by CamJN** [comment]
should be fixed in 1.0.6

**Comment by groyoh** [comment]
Thanks!
